Sources/SmileID/Classes/Networking/ServiceRunnable.swift

```diff
@@ -108,6 +111,32 @@ extension ServiceRunnable {
         headers.append(.timestamp(value: timestamp))
         headers.append(.sourceSDK(value: "iOS"))
         headers.append(.sourceSDKVersion(value: SmileID.version))
+        let timestamp = Date().toISO8601WithMilliseconds()
+        headers.append(.requestTimestamp(value: timestamp))
+
+        let encoder = JSONEncoder()
+        encoder.outputFormatting = [.sortedKeys, .withoutEscapingSlashes]
+        let selfieRequest = SelfieRequest(
+            selfieImage: selfieImage.data,
+            livenessImages: livenessImages.map { $0.data },
+            userId: userId,
+            partnerParams: partnerParams,
+            callbackUrl: callbackUrl?.nilIfEmpty(),
+            sandboxResult: sandboxResult,
+            allowNewEnroll: allowNewEnroll,
+            failureReason: failureReason,
+            metadata: metadata.items
+        )
+        do {
+            let payload = try encoder.encode(selfieRequest)
+            let requestMac = try SmileIDCryptoManager.shared.sign(
+                timestamp: timestamp,
+                headers: headers.toDictionary(),
+                payload: payload
+            )
+            headers.append(.requestMac(value: requestMac))
+        } catch { /* in case we can't add the security info the backend will deal with the enrollment */ }
```

Comment:
We might not really need a do-catch statement here since we are not handling any error in the catch block and since this is inside a throwable function we can just use an if-statement.

Reply:
Resolved in [f1c0a73](https://github.com/smileidentity/ios/pull/286/commits/f1c0a7342480f07515b317fc20349c7a7ddf0bd5)